src/components/car/CreateCarForm.js

```diff
@@ -0,0 +1,71 @@
+/* eslint-disable jsx-a11y/label-has-associated-control */
+// CarForm.js
+import React, { useState } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { createCar } from '../../redux/actions/carActions';
+import './CreateCarForm.css';
+
+function CarForm() {
```

Comment:
- [ ] When I try to add a car, I get this error:

![image](https://github.com/BHObserver/Rent-a-Car-frontend/assets/57421823/2c88c316-742e-4095-b965-fb873bdeeca7)

![image](https://github.com/BHObserver/Rent-a-Car-frontend/assets/57421823/7f04a08f-c495-458f-bcef-2ffb43b9b3bf)

![image](https://github.com/BHObserver/Rent-a-Car-frontend/assets/57421823/914da5c6-a57d-4ef9-a951-37532a2a694f)

Kindly, fix this bug so your app works as intended. 👍 

_note: Since I couldn't add any car, I couldn't review the delete functionality.  This will be taken a look at in the next review._